lib/bulk/common.js

```diff
@@ -1119,7 +1119,7 @@ class BulkOperationBase {
         callback,
         new BulkWriteError(
           toError({
-            message: 'write operation failed',
+            message: this.s.bulkResult.writeErrors[0].errmsg,
```

Comment:
What happens if `this.s.bulkResult.writeErrors[0]` or its `errmsg` doesn't exist? Let's check that the `errmsg` exists and use the default `write operation failed` message if not.